src/main/java/com/noroff/mefit/model/Profile.java

```diff
@@ -0,0 +1,83 @@
+package com.noroff.mefit.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.Size;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "profile", schema = "public")
+public class Profile {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Getter
+    @Setter
+    @Column(name = "id")
+    private Integer id;
+
+    @Getter @Setter
+    @Size(min = 1, max = 50)
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "user_id")
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    private User userId;
+
+//    @Getter @Setter
+//    @Size(min = 1, max = 50)
+//    @Column(name = "goal_id")
+//    private Integer goalId;
+
+//    @Getter @Setter
+//    @Size(min = 1, max = 50)
+//    @OneToOne(cascade = CascadeType.ALL)
+//    @JoinColumn(name = "address_id")
+//    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+//    private Address addressId;
+
+//    @Getter @Setter
+//    @Size(min = 1, max = 50)
+//    @OneToOne(cascade = CascadeType.ALL)
+//    @JoinColumn(name = "program_id")
+//    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+//    private Program programId;
+
+//    @Getter @Setter
+//    @Size(min = 1, max = 50)
+//    @OneToOne(cascade = CascadeType.ALL)
+//    @JoinColumn(name = "workout_id")
+//    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+//    private Workout workoutId;
+
+    @Getter @Setter
+    @Size(min = 1, max = 50)
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "set_id")
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    private Set setId;
+
+    @Getter @Setter
+    @Size(min = 1, max = 50)
```

Comment:
It does not make sense to have a 50 character long integer as max value in weight.
The maximum bodyweight a human can survive is around 450kg (heavies recorded 442kg).
```suggestion
    @Size(min = 1, max = 3)
```
If we want users to be able to display other values such as lbs or something else we should not limit the range to only 3. Then if you weigh over 453.6kg you reach the 4th value at approx. 1000lbs. In Ounces you would reach the 5th value.
```suggestion
    @Size(min = 1, max = 5)
```